### ISSUE TITLE
Add gravity options to responsive feature

### ISSUE THREAD

**Issue by askrzypczak** [cloudinary/cloudinary-vue#82]
## Explain your use case
… (A high level explanation of why you need this feature)

When using `responsive="fill"` on `CldImage`, the responsive size transformation in the generated url will bypass gravity options passed into the component. This is a shame, since out-of-the-box responsiveness and auto-gravity is one of the main draws to using cloudinary

## Describe the problem you’re trying to solve
… (A more technical view of what you’d like to accomplish, and how this feature will help you achieve it)

my code:
```
<cld-image
  responsive="fill"
  width="auto"
  height="auto"
  fetch-format="auto"
  quality="auto"
  dpr="auto"
  :public-id="xyz"
>
  <cld-transformation gravity="auto" />
</cld-image>
```
 it generates a url with this format: `.../upload/g_auto/c_fill,dpr_2.0,h_170,w_800/dpr_auto,q_auto/...`

if i try: 
```
<cld-image
  responsive="fill"
  width="auto"
  height="auto"
  fetch-format="auto"
  quality="auto"
  dpr="auto"
  gravity="auto"
  :public-id="xyz"
>
</cld-image>
```
then I get : `.../upload/c_fill,dpr_2.0,h_170,w_800/dpr_auto,g_auto,q_auto/...`

neither of these urls result in gravity:auto being taken into account in the reactive transformation.

If i change my url to have this format then i get gravity in my responsive feature: `.../upload/c_fill,dpr_2.0,h_170,w_800,g_auto/dpr_auto,q_auto/...`

You can quickly verify my assertions about the urls by looking at these images:
https://res.cloudinary.com/demo/image/upload/c_fill,dpr_2.0,h_170,w_800,g_auto/dpr_auto,q_auto/v1591726864/samples/people/kitchen-bar.jpg
https://res.cloudinary.com/demo/image/upload/g_auto/c_fill,dpr_2.0,h_170,w_800/dpr_auto,q_auto/v1591726864/samples/people/kitchen-bar.jpg
https://res.cloudinary.com/demo/image/upload/g_auto/c_fill,dpr_2.0,h_170,w_800/dpr_auto,g_auto,q_auto/v1591726864/samples/people/kitchen-bar.jpg

## Do you have a proposed solution?

add extra options to the responsive prop to allow gravity options to be included in the transformation

I was able to work around this limitation by specifying a transformation step to execute before the responsive transformation, but this required me to hardcode the bounding size of the image. It was sufficient for my use case but for cases where the responsive feature will handle large size changes then this will not be enough.

workaround: 
```
<cld-image
  responsive="fill"
  width="auto"
  height="auto"
  :public-id="xyz"
>
  <cld-transformation 
    width="800"
    height="300"
    crop="fill"
    gravity="auto"
  />
</cld-image>
```

**Comment by patrick-tolosa** [comment]
Thanks @askrzypczak  for your detailed report and suggestion.

We're looking into your suggestion, we'll update this thread as soon as we have an answer.
Thanks,
Patrick

**Comment by KazakovKirill** [comment]
Hello!
Is there any updates?

**Comment by patrick-tolosa** [comment]
Hello @KazakovKirill 

This change is on our road map. However, there is no ETA yet.
We'll update this thread once we have more information.

**Comment by mayashavin** [comment]
Fixed and Merged #117

**Comment by askrzypczak** [comment]
I can see that this was merged into 'master' but there doesnt seem to be a release yet. Where can i find information about the release schedule?

**Comment by patrick-tolosa** [comment]
Hey @askrzypczak - this is now available in version 1.2.0
